3p/integration.js

```diff
@@ -68,6 +68,7 @@ import {twitter} from './twitter';
 
 // 3P Ad Networks - please keep in alphabetic order
 import {_ping_} from '../ads/_ping_';
+import {_24smi} from '../ads/24smi';
```

Comment:
one minor request here: Since `_24smi` should come before `_ping` in alphabetic order. Can we move the `_ping_` (for test purpose) above the comment.  Should be
```
import {_ping_} from '../ads/_ping_';

// comments
import {_24smi}...
```